configs/freebind/freebind.yaml

```diff
@@ -0,0 +1,39 @@
+admin:
+  access_log_path: /tmp/admin_access.log
+  address:
+    socket_address: { address: 127.0.0.1, port_value: 9901 }
+
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address: { address: 192.168.42.1, port_value: 10000 }
+    freebind: true
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        config:
+          stat_prefix: ingress_http
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: local_service
+              domains: ["*"]
+              routes:
+              - match: { prefix: "/" }
+                route: { cluster: service_local }
+          http_filters:
+          - name: envoy.router
+  clusters:
+  - name: service_local
+    connect_timeout: 30s
+    type: STATIC
+    lb_policy: ROUND_ROBIN
+    hosts:
+    - socket_address:
+        address: 127.0.0.1
+        port_value: 10001
+#    upstream_bind_config:
```

Comment:
Do you mean for this to be commented out?

Reply:
Yes, it relates to my TODO on figuring out how to test this working with packets flowing end-to-end. I was mucking around with `iptables` today, but it seems that basic NATing isn't sufficient. I'll add an explicit TODO there as well.